test/dif_pe_examples/vp/vp-simple-age-predicate.json

```diff
@@ -0,0 +1,38 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://identity.foundation/presentation-exchange/submission/v1"
+  ],
+  "presentation_submission": {
+    "id": "accd5adf-1dbf-4ed9-9ba2-d687476126cb",
+    "definition_id": "31e2f0f1-6b70-411d-b239-56aed5321884",
+    "descriptor_map": [
+      {
+        "id": "867bfe7a-5b91-46b2-9ba4-70028b8d9cc8",
+        "format": "ldp_vp",
+        "path": "$.verifiableCredential[0]"
+      }
+    ]
+  },
+  "type": [
+    "VerifiablePresentation",
+    "PresentationSubmission"
+  ],
+  "verifiableCredential": [
+    {
+      "@context": [
+        "https://www.w3.org/2018/credentials/v1"
+      ],
+      "age": 19,
+      "credentialSchema": [
+        {
+          "id": "https://www.w3.org/TR/vc-data-model/#types"
+        }
+      ],
+      "credentialSubject": null,
+      "id": "2dc74354-e965-4883-be5e-bfec48bf60c7",
+      "issuer": "",
+      "type": "VerifiableCredential"
+    }
+  ]
+}
```

Comment:
We should have one more example where input descriptor describes 2 fields. But VP has 3 fields. One additional field.
In this case the 3rd field will not be selected for the submission.

Reply:
ok, working on it

Reply:
in presentationDefinition:

```
...
"fields": [
            {
              "path": [
                "$.age",
                "$.details.age"
              ],
              "filter": {
                "type": "integer",
                "minimum": 18
              },
              "predicate": "required"
            }, {
              "path": [
                "$.citizenship",
                "$.details.citizenship"
              ],
              "filter": {
                "type": "string",
                "enum": [
                  "eu",
                  "us",
                  "uk"
                ]
              },
              "predicate": "required"
            }
          ]
```

And in verifiablePresentation:

```
...
      "age": 19,
      "details": {
        "citizenship": "eu"
      },
      "birthPlace": "Maarssen",
```